content/id/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Plugin Jaringan
+content_template: templates/concept
+weight: 10
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+{{< warning >}}Fitur-fitur Alpha berubah dengan cepat. {{< /warning >}}
+
+Network plugins in Kubernetes come in a few flavors:
+_Plugin_ jaringan di Kubernet datang dalam beberapa varian:
```

Comment:
```suggestion
_Plugin_ jaringan di Kubernetes hadir dalam beberapa varian:
```

- Typo `Kubernet`
- We can change `datang` to `hadir` to remove confusion from the literal translation

Reply:
done